### PR TITLE
Update MaterialLib.py

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: pyaedt/modules/MaterialLib.py [hunk 1]
```diff
@@ -93,17 +92,18 @@ def _get_surface_materials(self):
...
     @aedt_exception_handler
     def checkifmaterialexists(self, mat):
-        """Check if a material Exists in AEDT. If it exists and it's not in material database it will populate it
+        """Check if a material exists in AEDT.
 
         Parameters
         ----------
         mat : str
-            material name
+            Name of the material. If the material exists and is not in the materials databse,
```
Comment (PipKat) [inline pyaedt/modules/MaterialLib.py:100]:
```suggestion
            Name of the material. If the material exists and is not in the materials database, 
```

File: pyaedt/modules/MaterialLib.py [hunk 2]
```diff
@@ -1,14 +1,10 @@
...
-Material Library Class
-----------------------------------------------------------------
-
-
-Description
-===========
-
-This class contains all the functionalities to create and edit materials
-
+This class contains all functionalities for creating and editing materials
```
Comment (PipKat) [inline pyaedt/modules/MaterialLib.py:2]:
```suggestion
This module contains all functionalities for creating and editing materials
```

Comment (PipKat) [inline pyaedt/modules/MaterialLib.py:2]:
```suggestion
This module contains the `Materials` class. 
```

File: pyaedt/modules/MaterialLib.py [hunk 3]
```diff
@@ -19,33 +15,35 @@
 import json
 
 class Materials(object):
-    """This class contains AEDT Materials database and methods for creating and manipulating materials
+    """Materials class.
+    
+    This class contains the AEDT materials database and methods for creating and manipulating materials.
```
Comment (PipKat) [inline pyaedt/modules/MaterialLib.py:20]:
```suggestion
    This class contains the AEDT materials database and all methods for creating and editing editing materials.
```

File: pyaedt/modules/MaterialLib.py [hunk 4]
```diff
@@ -1,14 +1,10 @@
...
-===========
-
-This class contains all the functionalities to create and edit materials
-
+This module contains the `Materials` class. 
 
+Parameters
+----------
+parent : str
+    Name of the parent AEDT application.
```
Comment (akaszynski) [inline pyaedt/modules/MaterialLib.py:7]: Parameters section should be in the class.

File: pyaedt/modules/MaterialLib.py [hunk 5]
```diff
@@ -19,33 +15,35 @@
 import json
 
 class Materials(object):
-    """This class contains AEDT Materials database and methods for creating and manipulating materials
+    """Materials class.
+    
+    This class contains the AEDT materials database and all methods for creating and editing editing materials.
```
Comment (akaszynski) [inline pyaedt/modules/MaterialLib.py:20]:
```suggestion
    This class contains the AEDT materials database and all methods for creating and editing materials.
```

File: pyaedt/modules/MaterialLib.py [hunk 6]
```diff
@@ -148,21 +150,21 @@ def check_thermal_modifier(self, mat):
...
+        the material. This method does not update the material.
+        
         Parameters
         ----------
         materialname : str
-            material name
-
-        props : dict
-            Optional material property dictionary. Default is `None`.
+            Bame of the material.
```
Comment (akaszynski) [inline pyaedt/modules/MaterialLib.py:161]:
```suggestion
            Name of the material.
```

File: pyaedt/modules/MaterialLib.py [hunk 7]
```diff
@@ -303,19 +311,19 @@ def add_material_sweep(self, swargs, matname):
...
+            Name of the material to duplicate.
         new_name : str
-            target name
+            Name for the copy of the material.
 
         Returns
         -------
         Material
-            new material object
+           Material object that waas created.
```
Comment (akaszynski) [inline pyaedt/modules/MaterialLib.py:326]:
```suggestion
            Material object that was created.
```

File: pyaedt/modules/MaterialLib.py [hunk 8]
```diff
@@ -367,18 +377,20 @@ def duplicate_surface_material(self, material, new_name):
...
-        """Remove Material
+        """Remove a material.
 
         Parameters
         ----------
         material : str
-            material name
-
+            Name of the material.
+        library : stri, optional
```
Comment (akaszynski) [inline pyaedt/modules/MaterialLib.py:386]:
```suggestion
        library : str, optional
```

File: pyaedt/modules/MaterialLib.py [hunk 9]
```diff
@@ -367,18 +377,20 @@ def duplicate_surface_material(self, material, new_name):
...
+            Name of the library containing this material.
+            The default is ``"Project"``.
 
         Returns
         -------
         bool
-            ``True`` if succeeded
+             ``True`` when successful, ``False`` when failed.
```
Comment (akaszynski) [inline pyaedt/modules/MaterialLib.py:393]:
```suggestion
            ``True`` when successful, ``False`` when failed.
```

File: pyaedt/modules/MaterialLib.py [hunk 10]
```diff
@@ -399,13 +411,14 @@ def remove_material(self, material, library="Project"):
 
     @property
     def conductors(self):
-        """Get List of conductors in material database
+        """Conductors in the material database.
```
Comment (akaszynski) [inline pyaedt/modules/MaterialLib.py:416]:
```suggestion
```

File: pyaedt/modules/MaterialLib.py [hunk 11]
```diff
@@ -455,18 +465,18 @@ def _aedmattolibrary(self, matname):
...
-        """
-        Save all materials to Json File
-
+        """Export all materials to a JSON file.
+        
         Parameters
         ----------
         full_json_path : str
-            full path to json file
+            Full path and name for the JSON file.
```
Comment (akaszynski) [inline pyaedt/modules/MaterialLib.py:473]:
```suggestion
            Full path to export the JSON file to.
```

**Review by akaszynski — APPROVED** [review]
Approved with minor suggestions.

**Pull request merged**